pkg/kine/prepared/db.go

```diff
@@ -0,0 +1,233 @@
+package prepared
```

Comment:
Should we move this file under generic?

Reply:
I want this to not be part of a driver so that if we have a new implementation (and we will, hopefully), then we can share this package.